lib/core/src/client/preview/start.js

```diff
@@ -350,13 +350,21 @@ export default function start(render, { decorateStory } = {}) {
     let currentExports = new Map();
     if (reqs) {
       reqs.forEach(req => {
-        req.keys().forEach(filename => {
-          const fileExports = req(filename);
-          currentExports.set(
-            fileExports,
-            typeof req.resolve === 'function' ? req.resolve(filename) : null
-          );
-        });
+        if (typeof req.keys === 'function') {
```

Comment:
I don't think you should need to change this. @shilman and I already covered off this use case when defining this function I think.